123_state_tasks.R

```diff
@@ -11,7 +12,7 @@ do_state_tasks <- function(oldest_active_sites, ...) {
     },
     #command = 'get_site_data(oldest_active_sites, target_name, parameter)'
     command = function(task_name, ...){
-        sprintf("get_site_data(oldest_active_sites, I('%s'), parameter)", task_name)
+        sprintf("get_site_data(I('1_fetch/tmp/inventory_%s.tsv'), parameter)", task_name)
```

Comment:
Wrapping this in `I()` will cause the target to skip being rebuilt when the contents of the inventory file change, which is not what you want to have happen here (you _do_ want the build). I think you just need to remove the `I()` for `sprintf("get_site_data(state_info_file='1_fetch/tmp/inventory_%s.tsv', parameter=parameter)", task_name)`